pkg/storaged/warnings.jsx

```diff
@@ -75,26 +84,26 @@ export function find_warnings(client) {
         const vdo = content_block ? client.legacy_vdo_overlay.find_by_backing_block(content_block) : null;
         const stratis_bdev = client.blocks_stratis_blockdev[content_path];
 
-        if (fsys && fsys.Size && (lvol.Size - fsys.Size - crypto_overhead) > vgroup.ExtentSize && fsys.Resize) {
+        if (fsys && fsys.Size && (size - fsys.Size - crypto_overhead) > min_change && fsys.Resize) {
             enter_warning(path, {
                 warning: "unused-space",
-                volume_size: lvol.Size - crypto_overhead,
+                volume_size: size - crypto_overhead,
                 content_size: fsys.Size
             });
         }
 
-        if (vdo && (lvol.Size - vdo.physical_size - crypto_overhead) > vgroup.ExtentSize) {
+        if (vdo && (size - vdo.physical_size - crypto_overhead) > min_change) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19385-20230928-111839-72431268-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)